### DISCUSSION TITLE
Minimum system requirements in the cloud

### DISCUSSION THREAD

**Discussion by MSoliven** [microrealestate/microrealestate#173] [General]
Thanks for starting this fantastic project with lots of potential. I own a rental property and would like to contribute/ customize this by adding payment gateways, etc.
I understand that this is an alpha version, but if I were to deploy this on the cloud (e.g., Cloud Run, AWS Lightsail) as-is, what do you think are the minimum memory and CPU requirements for each node with very light usage (say, 5 landlords and 50 tenants)? I'm just trying to understand how much it will cost me to operate this web application. Thanks.

**Comment by camelaissani** [comment]
Hi @MSoliven currently I run the full app with Mongo and Redis perfectly on a VPS with the following specs:

**CPU**
2 vCores	
**RAM**
2 Gb
**Disk**
80 Gb but 40Gb should be enough

**Reply by MSoliven** [reply]
Thank you, @camelaissani. I have this running on a Debian AWS Lightsail instance with similar specs. I assume you have HTTPS enabled on your VPS. If so, would you mind sharing? I just created a different thread for the topic.
